src/gatsby-node.js

```diff
@@ -38,6 +40,7 @@ export const sourceNodes = async (gatsbyContext, pluginOptions) => {
       ...feature,
       id: createNodeId([name, feature?.id].filter(Boolean).join(' ')),
       featureId: feature?.id,
+      polylabel: polylabel(feature.geometry.coordinates, 0.1),
```

Comment:
Can you check the GeoJSON spec to ensure `geometry.coordinates` is always provided? What happens when the feature is a line? Or a point? Does polylabel work for those GeoJSON types?

GeoJSON RFP here: https://tools.ietf.org/html/rfc7946

Reply:
I believe any GeoJSON type has `geometry.coordinates`.

See section 3.1 in the linked RFP: https://tools.ietf.org/html/rfc7946#section-3.1

> A GeoJSON Geometry object of any type other than 
> "GeometryCollection" has a member with the name "coordinates".

Reply:
Yeah, just tested on lines and points, and it wont work on those types. I've changed to only provide polylabel if the feature geometry type is a `Polygon`